shared/src/business/entities/EntityConstants.ts

```diff
@@ -1733,3 +1733,8 @@ export const STATUS_REPORT_ORDER_OPTIONS = {
     restored: 'restoredToGeneralDocket',
   },
 };
+
+export const TROUBLESHOOTING_INFO = {
+  APP_SUPPORT_EMAIL: 'dawson.support@ustaxcourt.gov',
```

Comment:
Pulling this out as a constant since I see it being used all over the place. This small change constitutes another non-trivial portion of the diff.